pyproject.toml

```
@@ -24,7 +24,7 @@ django = "~2.2"
 django-babel = "^0"
 django-countries = "^5.3"
 django-filter = "^2.2"
-django-graphql-jwt = "<=0.2.1" # https://github.com/mirumee/saleor/issues/4652
+django-graphql-jwt = "^0.3.0" # https://github.com/mirumee/saleor/issues/4652
```

Obviously, the comment is out of date from that change
```suggestion
django-graphql-jwt = "^0.3.0"
```